src/DinosaurHydrator/DinosaurHydrator.php

```diff
@@ -53,7 +53,7 @@ public static function getDino(PDO $db, int $id): ?Dinosaur
     public static function getSearchedDinos(PDO $db, string $search, string $filter = ''): Array
     {
         // Prepares (/stores) the criteria for data we want to retrieve from the db
-        if (trim($filter) !== ''){
+        if (trim($filter) !== '0'){
```

Comment:
Ah yes sorry - we added this so that when you use the toggle if the user clicks filter by food type instead of a type it doesn't filter. Will try and set the value to empty string so it works for both :)